content/next/index.md

```diff
@@ -91,12 +96,14 @@ The description is a short summary of the code changes, e.g., _fix: array parsin
 1. A footer of one or more lines MAY be provided one blank line after the body. The footer MUST contain meta-information
 about the commit, e.g., related pull-requests, reviewers, breaking changes, with one piece of meta-information
 per-line.
-1. Breaking changes MUST be indicated at the very beginning of the body section, or at the beginning of a line in the footer section. A breaking change MUST consist of the uppercase text BREAKING CHANGE, followed by a colon and a space.
-1. A description MUST be provided after the `BREAKING CHANGE: `, describing what has changed about the API, e.g., _BREAKING CHANGE: environment variables now take precedence over config files._
+1. Breaking changes MUST be indicated in the type/scope prefix of a commit, or at the beginning of its optional body or footer section.
+1. If included in the body or footer, a breaking change MUST consist of the uppercase text BREAKING CHANGE, followed by a colon, space, and description, e.g.,
```

Comment:
maybe we should call out the fact that a reason for using `BREAKING CHANGE` would be that you wish to provide a longer description of the breaking change.